src/java.desktop/macosx/classes/com/apple/laf/AquaButtonUI.java

```diff
@@ -305,7 +305,15 @@ public void paint(final Graphics g, final JComponent c) {
         }
 
         // performs icon and text rect calculations
-        final String text = layoutAndGetText(g, b, aquaBorder, i, viewRect, iconRect, textRect);
+        final String text;
+        final View v = (View)c.getClientProperty(BasicHTML.propertyKey);
+        if (v != null) {
+            // use zero insets for view since layout only handles text calculations
+            text = layoutAndGetText(g, b, aquaBorder, new
+                    Insets(0,0,0,0), viewRect, iconRect, textRect);
```

Comment:
Please do not break object instantiation: always keep `new` and class name on the same line.